homeassistant/components/sleepiq/__init__.py

```diff
@@ -89,6 +90,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     coordinator = SleepIQDataUpdateCoordinator(hass, gateway, email)
     pause_coordinator = SleepIQPauseUpdateCoordinator(hass, gateway, email)
+    actuator_coordinator = SleepIQActuatorUpdateCoordinator(hass, gateway, email)
```

Comment:
You can use the `SleepIQDataUpdateCoordinator` for this as they have the same update interval

Reply:
The goal was to avoid making the extra API call to get the values for actuators if those entities are disabled, but I'm not sure if it actually works that way. If not, then definitely no reason for an extra coordinator.

Reply:
The coordinator would still update even if there were no entities.  If there are no physical actuators then at least it won't make any API call

Reply:
If the coordinator is updating and there are no entities subscribed that's a bug